config/routes.rb

```diff
@@ -1,6 +1,22 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  apipie
+  devise_for :users, controllers: {
+    registrations: 'registrations',
+    sessions: 'sessions',
+    passwords: 'passwords'
+  }
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  devise_scope :user do
+    post 'users/sign_up', to: 'registrations#create'
+    post 'users/sign_in', to: 'sessions#create'
+    delete 'users/sign_out', to: 'sessions#destroy'
+    post 'users/password', to: 'passwords#create'
+    put 'users/password', to: 'passwords#update'
+  end
+
+  resources :reservations
+  resources :users
+  resources :vespas
+  resources :comments
+  resources :likes
```

Comment:
- [ ]  OPTIONAL: Nice routes and nice use of the `#resources` method. However, you are generating more routes than needed.

## Explanation

By using the `only:` keyword parameter, you can generate only the routes that you are using.

[Look](https://launchschool.com/books/demystifying_rails/read/routes_and_resources):

![image](https://github.com/djo1975/Program-Capstone/assets/57421823/8ab1821e-fc2f-4cab-a114-16c358cc6fae)


## Example

For the likes controller, I can see that you are only using index, create, show, update, and destroy (no edit or new).

So:

```ruby
resources :likes, only: [:index, :create, :show, :update, :destroy]

```

That way, your routes will be more consistent, and only the ones needed will be generated💪